tests/duplicate-dep/metadata.json

```diff
@@ -57,13 +57,9 @@
     }
   ],
   "requirements": [
-    {
-      "name": "pe",
-      "version_requirement": ">= 3.2.0 < 3.4.0"
-    },
     {
       "name": "puppet",
-      "version_requirement": "3.x"
+      "version_requirement": "4.10.0"
```

Comment:
To prevent having to change this constantly as the minimum supported version is updated, I suggest setting it to the current version, `5.5.1`.